.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "julia.environmentPath": "c:\\Users\\ngudat\\.julia\\dev\\DataFrames.jl"
```

Comment:
please remove this file

Reply:
Thanks - goes to show how well my switch to VSCode is going 😄
The file showed as "untracked" in VSCode but apparently still was committed...

Reply:
Have deleted!